README.md

```diff
@@ -1,50 +1,69 @@
 # BigchainDB Whitepaper
 
+![repo banner](media/repo-banner@2x.png)
+
 This repository contains the source (LaTeX) for the BigchainDB whitepaper.
 
+---
+
+**The most recent PDF version of the whitepaper can always be retrieved from [bigchaindb.com/whitepaper](https://www.bigchaindb.com/whitepaper)**
+
+---
+
 If you're looking for the main BigchainDB repository, it's at [bigchaindb/bigchaindb](https://github.com/bigchaindb/bigchaindb).
 
+And if you're really here to get a glimpse of the exciting world of LaTeX-based file generation, this repo is for you. 
 
-## Prerequisites for Using Bash Script or Make
+## Prerequisites for generating the PDF
 
-1. [TeX Live](https://www.tug.org/texlive/) or similar
-2. pdftk (can be installed using `apt-get install pdftk`)
+Build requires two binaries:
 
+- `pdflatex`
+- `bibtex`
 
-## Using the Bash Script to Generate the PDF
+### Using Docker
```

Comment:
This sub-subsection about Using Docker is really how to use Docker Compose to build the whitepaper. It's not about prerequisites, so it shouldn't be part of the subsection about prerequisites. In fact, if they want to use Docker Compose, there are prerequisites for that, i.e. they must have Docker Compose installed.

Maybe move this stuff about how to use Docker Compose to the very end, as an alternative to the bash script (i.e. another way to generate the whitepaper).